src/models/async/JobStatus.ts

```diff
@@ -0,0 +1,5 @@
+export enum JobStatus {
```

Comment:
status is not specific to `async` right? I think it should be directly in the `/models/` namespace